packages/features/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@standardnotes/features",
-  "version": "1.2.3",
+  "version": "1.2.4",
```

Comment:
can we change this to 1.3.0?

@vardan-arm feel free to merge since the PR is approved

Reply:
Done. And got it)